butterfree/transform/features/timestamp_feature.py

```diff
@@ -41,6 +41,7 @@ class TimestampFeature(Feature):
 
     def __init__(
         self,
+        dtype: Optional[DataType] = DataType.TIMESTAMP,
```

Comment:
Should be?
```suggestion
        dtype: Optional[DataType] = DataType.TIMESTAMP_NTZ,
```

Reply:
Hey @albjoaov !! I set TIMESTAMP as default to migrate our feature sets smoothly!